flexmeasures/ui/utils/view_utils.py

```diff
@@ -86,14 +87,19 @@ def render_flexmeasures_template(html_filename: str, **variables):
         options["downloadFileName"] = f"asset-{asset.id}-{asset.name}"
     variables["chart_options"] = json.dumps(options)
 
-    variables["menu_logo"] = current_app.config.get("FLEXMEASURES_MENU_LOGO_PATH")
+    account: Account | None = (
+        current_user.account if current_user.is_authenticated else None
+    )
+
+    # check if user has logo_url set
+    if account and account.logo_url:
```

Comment:
Account would be None only when visiting the login page, right? Perhaps in addition to a per-account logo we could have a server-default logo different to the FlexMeasures logo. Just an idea for a potential follow-up ticket.

Reply:
We do have that, it's the config setting `FLEXMEASURES_MENU_LOGO_PATH`

Reply:
Oh right, of course.

Reply:
But indeed, the user experience of white-labeling will be not 100% perfect, as it only begins after authentication.

Probably okay for now. I guess the real way out would be to offer partners their own subdomain and checking the subdomain used in our code, which then can be linked to the logo of an account with that name ...  Too much work for now.